.github/workflows/CI.yml

```
@@ -0,0 +1,74 @@
+name: CI
+# Run on master, any tag or any pull request
+on:
+  push:
+    branches:
+      - master
+      - staging
+      - trying
+    tags: '*'
+  schedule: 
+    - cron: '0 2 * * *'  # Daily at 2 a.m. UTC
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.version == 'nightly' }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - 1
```

Depending on what we want, we can put the 1.0 here, or just that single job in the include
```suggestion
          - "1.0"  # LTS
          - "1"     # Latest Release
```